ksqldb-parser/src/main/java/io/confluent/ksql/parser/tree/DropConnector.java

```diff
@@ -41,18 +53,19 @@ public boolean equals(final Object o) {
       return false;
     }
     final DropConnector that = (DropConnector) o;
-    return Objects.equals(connectorName, that.connectorName);
+    return Objects.equals(connectorName, that.connectorName) && ifExists == that.ifExists;
   }
 
   @Override
   public int hashCode() {
-    return Objects.hash(connectorName);
+    return Objects.hash(ifExists, connectorName);
   }
 
   @Override
   public String toString() {
-    return "DropConnector{"
-        + "connectorName='" + connectorName + '\''
-        + '}';
+    return toStringHelper(this)
```

Comment:
Is there a reason why we're switching to using this toStringHelper function now? There seems to be a mix of using this method vs defining out manually what the string representation looks like for different AstNodes.

Reply:
Not particularly, just thought it looked neater especially now that there is more than one field.